scripts/docs/sort-glossary.mjs

```diff
@@ -0,0 +1,139 @@
+import { readFile, writeFile } from 'node:fs';
+import { resolve } from 'node:path';
+
+import c from 'chalk';
+import { remark } from 'remark';
+import gfm from 'remark-gfm';
+import parse from 'remark-parse';
+import stringify from 'remark-stringify';
+import { diffTrimmedLines } from 'diff';
```

Comment:
we can alphabetize by imported thing, but what you're looking for doesn't work concurrently with import/order for eslint

Reply:
> what you're looking for doesn't work concurrently with import/order for eslint

I am not sure what are you talking about. :thinking: Should [`.eslintrc.js` (L35-L41)](https://github.com/romcal/romcal/blob/6eeddbb149ceae93eb3c2f8a5cd618c7fe2ae5f3/.eslintrc.js#L35-L41) not take care of the sorting? It works in my other projects. :thinking:

Reply:
not in the way you're talking about. if the names of the imports match the imported module name, it would sort the same, but sorting by package name is different from sorting by imported variable

Reply:
I don’t know your preferences, but I prefer sorting imports by package name, not by imported variable/object. But I see you have already done in [here](https://github.com/romcal/romcal/blob/93e9a26aa79d9a804d19b6061d0ab93f306c7f08/.eslintrc.js#L30-L33)–thanks! :pray: